src_c/mixer.c

```diff
@@ -34,6 +34,16 @@
 
 #define PyBUF_HAS_FLAG(f, F) (((f) & (F)) == (F))
 
+static const char *sound_not_initialised_errmsg =
```

Comment:
Not sure why this needs a variable defined, since this is only used once. You could just put the string in the `PyErr_SetString` call. Clang will format it nicely for you as well.

Reply:
Good point, that has been changed